### PR TITLE
add support of ALTER RESOURCE COST for Oracle

### LINKED ISSUE #20397: support parse ALTER RESOURCE COST
Hi community,

This issue is for #20305.
ref to #6480

## Background
ShardingSphere parser engine helps users parse a SQL to get the AST (Abstract Syntax Tree) and visit this tree to get SQLStatement (Java Object). At present, this parser engine can handle SQLs for `MySQL`, `PostgreSQL`, `SQLServer`, `openGauss` and `Oracle`, which means we have to understand different database dialect SQLs.
More details: 
https://shardingsphere.apache.org/document/current/en/reference/sharding/parse/

## Task
This issue is to support more sql parse. 

- [ ] ALTER RESOURCE COST [doc](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/ALTER-RESOURCE-COST.html#GUID-92DCB41E-5113-4722-8A54-E90E1AE7DB54)

## Process
1. Compare SQL definitions in [Oficial SQL Doc](https://docs.oracle.com/en/database/oracle/oracle-database/18/lnpls/plsql-language-elements.html#GUID-285DFAEE-1A34-4248-BA4B-2591476A0C3C) and [ShardingSphere SQL Doc](https://github.com/apache/shardingsphere/tree/master/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle).
2. If there is any difference in ShardingSphere SQL Doc, please correct them by referring to the Official SQL Doc.
3. Run `mvn install the current_file_module`.
4. Check whether there are any exceptions. If indeed, please fix them. (`Especially xxxVisitor.class`).
5. Add new corresponding SQL case in SQL Cases and expected parsed result in Expected Statment XML.
6. Run SQLParserParameterizedTest to make sure no exceptions.

## Relevant Skills
1. Master JAVA language
2. Have a basic understanding of Antlr g4 file
3. Be familiar with Oracle SQLs

### PR DESCRIPTION
**Pull request by tianhao960**
Fixes #20397 .

Changes proposed in this pull request:
  - add support of ALTER RESOURCE COST for Oracle
  -
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have passed maven check: `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/OracleKeyword.g4 [hunk 1]
```diff
@@ -3062,4 +3062,20 @@ OTHERS
...
+    : C O N N E C T UL_ T I M E
+    ;
+
+LOGICAL_READS_PER_SESSION
+    : L O G I C A L UL_ R E A D S UL_ P E R UL_ S E S S I O N
+    ;
+
+PRIVATE_SGA
+    : P R I V A T E UL_ S G A
     ;
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/OracleKeyword.g4:3081]: Please add a new line at the end of file.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dal/OracleAlterResourceCostStatement.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.DALStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
+
+/**
+ * Oracle alter resource statement.
+ */
+@Getter
+@Setter
+@ToString(callSuper = true)
+public class OracleAlterResourceCostStatement extends AbstractSQLStatement implements DALStatement, OracleStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dal/OracleAlterResourceCostStatement.java:33]: Please add final for OracleAlterResourceCostStatement.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dal/OracleAlterResourceCostStatement.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.DALStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
+
+/**
+ * Oracle alter resource statement.
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dal/OracleAlterResourceCostStatement.java:28]: Oracle alter resource cost statement may be better.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dal/OracleAlterResourceCostStatement.java (hunk 3 above)
Comment (tianhao960) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/dal/OracleAlterResourceCostStatement.java]: hi, have modify code as review advice.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**